tests/config/config.py

```diff
@@ -1,8 +1,4 @@
-# TODO DM-17597: Use the astrometrySourceSelector until we redo the refcats and metrics.
-# Once we do that, we can try using the sourceSelector defaults and recompute all
-# the metrics to match, though it may result in problems due to not having enough sources
-# because the testdata is generally fainter than the HSC/LSST data that the defaults
-# are now designed for.
-config.sourceSelector.name = "astrometry"
-config.sourceSelector["astrometry"].sourceFluxType = "Calib"
-config.sourceSelector["astrometry"].badFlags.extend(["slot_Shape_flag", "base_PixelFlags_flag_interpolated"])
+# Config overrides for jointcal tests.
+
+# jointcal test metrics were made including unresolved sources.
+config.sourceSelector['science'].doUnresolved = False
```

Comment:
No, for a couple of reasons: 1) with the test data I have, restricting to stars only would mean too few sources on some detectors, and 2) I'd have to redo all of the numbers in the tests (since everything would change), which isn't worth the time. The tests are meant to confirm that jointcal runs and produces sensible results; testdata_jointcal doesn't have enough data in it to serve as a test of a full fit. That's what the RC datasets are for.